draft-ietf-quic-multipath.md

```diff
@@ -1197,12 +1197,22 @@ Path Identifier:
 Note that, other than for the NEW_CONNECTION_ID frame of {{Section 19.15 of QUIC-TRANSPORT}},
 the sequence number applies on a per-path context.
 This means different connection IDs on different paths may have the same
-sequence number value. Respectively, the Retire Prior To field indicates which connection IDs
-should be retired for the path with the Path ID in the Path Identifier field.
+sequence number value. However, the value of the Connection ID MUST be unique
+across all paths.
```

Comment:
I don't think we need normative language here as this is already specified in RFC9000. Let's not specify this again.

Also note that 3.5 already says this:

```
As further specified in [Section 5.1](https://rfc-editor.org/rfc/rfc9000#section-5.1) of [[QUIC-TRANSPORT](https://quicwg.org/multipath/draft-ietf-quic-multipath.html#QUIC-TRANSPORT)] connection IDs cannot be issued more than once on the same connection and therefore are unique for the scope of the connection, regardless of the associated Path ID.
```

We can repeat it here (but not normatively).

Reply:
OK, removed the last sentence of that paragraph.